webpack/JobWizard/steps/HostsAndInputs/index.js

```diff
@@ -26,12 +33,34 @@ import {
   hostMethods,
   hostsController,
   hostQuerySearchID,
+  HOSTS_API,
+  HOSTS_TO_PREVIEW_AMOUNT,
 } from '../../JobWizardConstants';
 import { WizardTitle } from '../form/WizardTitle';
-import { HostSearch } from './HostSearch';
 import { SelectAPI } from './SelectAPI';
 import { SelectGQL } from './SelectGQL';
 
+const buildQuery = (selected, search) => {
+  const { hosts, hostCollections, hostGroups } = selected;
+  const hostsSearch = `(name = ${hosts
+    .map(({ id }) => id)
+    .join(' or name = ')})`;
+  const hostCollectionsSearch = `(host_collection_id = ${hostCollections
+    .map(({ id }) => id)
+    .join(' or host_collection_id = ')})`;
+  const hostGroupsSearch = `(hostgroup = ${hostGroups
+    .map(({ id }) => id)
+    .join(' or hostgroup = ')})`;
+  return [
```

Comment:
Not sure about this part as i dont have hostgroups

Reply:
It would be probably better to have this translate to a `hostgroup.id IN (...)`. In scoped search's query language that would be something along the lines of `(hostgroup.id ^ (${hostGroups.map(({ id }) => id).join(','))}))`

Reply:
Should I do this for the others as well? `host_collection_id` and `name`